Contents/Code/__init__.py

```diff
@@ -24,6 +25,36 @@ def Dict(var, *arg, **kwarg):  #Avoid TypeError: argument of type 'NoneType' is
     else:  return kwarg['default'] if kwarg and 'default' in kwarg else ""   # Allow Dict(var, tvdbid).isdigit() for example
   return kwarg['default'] if var in (None, '', 'N/A', 'null') and kwarg and 'default' in kwarg else "" if var in (None, '', 'N/A', 'null') else var
 
+### Used to Convert Crowd Sourced Video Titles to Title Case from Sentence Case
+def uppercase_regex(a):
+    return a.group(1) + a.group(2).upper()
+
+def titlecase(input_string):
+    return re.sub("(^|\s)(\S)", uppercase_regex, input_string)
+
+### These calls use DeArrow Created By Ajay Ramachandran to Obtain a Crowd Sourced Video Title
+def DeArrow(video_id):
+  api_url = 'https://sponsor.ajay.app'
+
+  hash = hashlib.sha256(video_id.encode('ascii')).hexdigest()
+  
+  # DeArrow API recommends using first 4 hash characters.
+  #url = f'{api_url}/api/branding/{hash[:4]}'
+  url = '{api_url}/api/branding/{hash}'.format(api_url = api_url, hash = hash[:4])
+
+  try:
+    data_json = {}
+    data_json = JSON.ObjectFromURL(url)
+  except:
+    crowd_sourced_title = ''
+
+  try:
+    crowd_sourced_title = titlecase(Dict(data_json[video_id]['titles'][0], 'title'))
```

Comment:
```suggestion
    first_title = video_id]['titles'][0]
    crowd_sourced_title = titlecase(Dict(first_title, 'title')) if Dict(first_title, 'votes') > 0 or Dict(first_title, 'locked')
```

Reply:
Eliminates untrusted titles from being used (in the extension these appear in the voting menu only, but not on videos themselves)

Reply:
Thank you! I merged your code with the suggested changes from @ZeroQI and refreshed the pull request